src/db.c

```diff
@@ -501,6 +509,27 @@ int dbGenericDelete(serverDb *db, robj *key, int async, int flags) {
     return dbGenericDeleteWithDictIndex(db, key, async, flags, dict_index);
 }
 
+/* Checks if the object is a hash object with volatile items and adds it to the hash field expiry kvstore */
+void dbTrackKeyWithVolaItemsIfNeeded(serverDb *db, robj *o) {
```

Comment:
nit: why are we becoming "Scrooge McDuck" on the name? in order to save txt file size due to symbols? 
I would name it `dbTrackKeyWithVolatileItemsIfNeeded`